deployments/liqo/values.yaml

```diff
@@ -35,6 +35,10 @@ controllerManager:
     # This feature is suggested to be enabled when consumer-side enforcement is not sufficient.
     # It has the same tradeoffs of resource quotas (i.e, it requires all offloaded pods to have resource limits set).
     enableResourceEnforcement: false
+    # -- the threshold (in percentage) of resources quantity variation which triggers a ResourceOffer update. Leave it empty to use the standard local resource monitor
+    offerUpdateThresholdPercentage: ""
+    # -- The address of an external resource monitor service, overriding the default resource computation logic based on the percentage of available resources.
+    externalMonitorAddress: ""
```

Comment:
Would move these flags close to `resourceSharingPercentage`, as conceptually related.